### PR TITLE
Style share btn overlapping text

### LINKED ISSUE #663: Transaction details overlap the share transaction button
## Bug description

A long transaction description, i.e. an on-chain rejection, overlaps the share button.

## Environment
 - Browser: Chrome
 - Wallet: MetaMask
 - Chain: Ethereum mainnet

## Steps to reproduce
 1. Create a transaction then a rejection for it. Open the details and observe the overlapping text.

## Expected result

The text should break before the share button.

## Obtained result

The text overlaps the button

## Screenshots

image

### PR DESCRIPTION
**Pull request by DiogoSoaress**
## What it solves

Resolves #663

## How this PR fixes it
Assigns some margin right to the copy

## How to test it
1. Expand a "On-chain rejection" transaction
2. Observe the copy that goes like "_This is an on-chain rejection that didn't send any funds. (...)_" to never stay under the Share button in different viewport widths

## Analytics changes
None

## Screenshots
Screenshot 2022-09-29 at 11 17 31

### CODE REVIEW

File: src/components/transactions/TxDetails/TxData/Rejection/index.tsx [hunk 1]
```diff
@@ -19,10 +19,9 @@ const RejectionTxInfo = ({ nonce, isTxExecuted }: Props) => {
 
   return (
     <>
-      <Typography>{message}</Typography>
+      <Typography mr={2}>{message}</Typography>
       {!isTxExecuted && (
-        <>
-          <br />
+        <Box mt={2} sx={{ width: 'fit-content' }}>
```
Comment (usame-algan) [inline src/components/transactions/TxDetails/TxData/Rejection/index.tsx:24]: What does the `fit-content` do? In Chrome it looks the same with and without.

Comment (DiogoSoaress) [reply src/components/transactions/TxDetails/TxData/Rejection/index.tsx:24]: To not have the link occupying the whole width

Comment (DiogoSoaress) [reply src/components/transactions/TxDetails/TxData/Rejection/index.tsx:24]: Addresses all this clickable area

Screenshot 2022-09-29 at 11 17 31

Comment (DiogoSoaress) [reply src/components/transactions/TxDetails/TxData/Rejection/index.tsx:24]: Addresses all this clickable area

Screenshot 2022-09-29 at 12 04 53

Comment (usame-algan) [reply src/components/transactions/TxDetails/TxData/Rejection/index.tsx:24]: Ah that makes sense, thanks!

**Review by usame-algan — APPROVED** [review]
👍

**Pull request merged**